### DISCUSSION TITLE
Can't get icons to load

### DISCUSSION THREAD

**Discussion by ghost** [vercel/next.js#13537] [Help]
I have a project: https://github.com/sc4224/typescript-graphql

I currently have argon-design-system-react installed as a node module: https://github.com/creativetimofficial/argon-design-system-react

As part of setup, in my index.tsx file I did:
`import 'argon-design-system-react/src/assets/vendor/nucleo/css/nucleo.css';`
`import 'argon-design-system-react/src/assets/vendor/font-awesome/css/font-awesome.min.css';`
`import 'argon-design-system-react/src/assets/scss/argon-design-system-react.scss';`

The good part is that the scss styles and fonts propagate. The bad news is that icons do not load.

Here is a picture of what I mean:
https://user-images.githubusercontent.com/30026659/82752743-c9b5c200-9df2-11ea-92a5-868091bb6d03.png

As you can see the scss shows up but the icon is replaced by a square placeholder.

In my project I setup my next.config.js file to be as follows:
```
const withFonts = require('next-fonts');
const withSass = require('@zeit/next-sass');
const withCSS = require('@zeit/next-css');

module.exports = withFonts(withCSS(withSass({
  enableSvg: true,
  webpack(config, options) {
    config.module.rules.push({
      test: /\.graphql$/,
      exclude: /node_modules/,
      use: [options.defaultLoaders.babel, { loader: 'graphql-let/loader' }],
    })

    config.module.rules.push({
      test: /\.graphqls$/,
      exclude: /node_modules/,
      use: ['graphql-tag/loader', 'graphql-let/schema/loader'],
    })

    config.module.rules.push({
      test: /\.(eot|woff|woff2|ttf|svg|png|jpg|gif)$/,
      use: {
        loader: 'url-loader',
        options: {
          limit: 100000,
          name: '[name].[ext]'
        }
      }
    })

    return config
  },
})));
```

Is there anything that I'm doing wrong in getting the icons to load?

**Comment by Rashmi-278** [comment]
I had similar error with now-ui-react kit 

here's how I resolved it. 

It's a loader issue. Add next.config.js file

```

// next.config.js 

const withPlugins = require("next-compose-plugins");
const withImages = require("next-images");
const withSass = require("@zeit/next-sass");
const withCSS = require("@zeit/next-css");
const withFonts = require("next-fonts");
const webpack = require("webpack");
const path = require("path");

module.exports = withFonts(
  withCSS(
    withImages(
      withSass({
        webpack(config, options) {
          config.module.rules.push({
            test: /\.(eot|ttf|woff|woff2)$/,
            use: {
              loader: "url-loader",
            },
          });
          config.resolve.modules.push(path.resolve("./"));
          return config;
        },
        sassOptions: {
          includePaths: [path.join(__dirname, 'styles')],
        }
      })
    )
  )
);
```

package info ,

 "@zeit/next-css": "^1.0.1",
    "@zeit/next-sass": "^1.0.1",
    "next-fonts": "^1.4.0",
    "node-sass": "^4.14.1",
    "webpack": "^4.44.1",
    "next-compose-plugins": "^2.2.0",
    "next-images": "^1.3.1",
